kernel/src/main/scala/cats/kernel/std/stream.scala

```diff
@@ -0,0 +1,58 @@
+package cats.kernel
+package std
+
+import cats.kernel.std.util.StaticMethods
+
+package object stream extends StreamInstances
+
+trait StreamInstances extends StreamInstances1 {
+  implicit def streamOrder[A: Order] = new StreamOrder[A]
```

Comment:
I think we should add explicit return types (`Order[Stream[A]]` as opposed to `StreamOrder[A]`) here and in other places in this file. I was thinking that we had a compiler flag that was enforcing this. Is it turned off in cats-kernel?